### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by krzychu124**
Updated copyright and year

### CODE REVIEW

File: TLM/TMPE.API/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 // associated with an assembly.
 [assembly: AssemblyTitle("TMPE.API")]
 [assembly: AssemblyProduct("TMPE.API")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("API assembly to help integration of other Cities: Skylines modifications with TMPE")]
```
Comment (kianzarrin) [inline TLM/TMPE.API/Properties/AssemblyInfo.cs:10]: I have a feeling that all this is for nuget package creation.

Comment (aubergine10) [reply TLM/TMPE.API/Properties/AssemblyInfo.cs:10]: wb!

**Review by aubergine10 — APPROVED** [review]
👍

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015 Svetlozar
+Copyright (c) 2019-2022 CitiesSkylinesMods/TMPE https://tmpe.me/
```
[the remark this answers is not in the document]
Comment (krzychu124) [reply LICENSE:4]: Hehe, I noticed it's slightly outdated.

File: TLM/TMPE.API/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (krzychu124) [reply TLM/TMPE.API/Properties/AssemblyInfo.cs:10]: It can be filled manually. Some day™️  we will release it as nuget package

**Comment by aubergine10** [comment]
looks like there's something wrong with teh CI server

**Review by DaEgi01 — APPROVED** [review]

**Comment by krzychu124** [comment]
> looks like there's something wrong with teh CI server

Yeah, uploading zip Artifacts is failing. I turned off that feature for now. Will check later if works

**Pull request merged**